sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/json/JsonScan.scala

```diff
@@ -91,7 +91,7 @@ case class JsonScan(
 
   override def hashCode(): Int = super.hashCode()
 
-  override def description(): String = {
-    super.description() + ", PushedFilters: " + pushedFilters.mkString("[", ", ", "]")
+  override def getMetaData(): Map[String, String] = {
+    super.getMetaData() ++ Map("PushedFilters" -> pushedFilters.mkString("[", ", ", "]"))
```

Comment:
Not sure why JSON was different than the others but made it the same by just providing metadata. Updated related UT below to accommodate.